markets/perps-market/contracts/interfaces/IAsyncOrderSettlementPythModule.sol

```diff
@@ -52,8 +52,6 @@ interface IAsyncOrderSettlementPythModule {
         int256 pnl;
         uint256 chargedInterest;
         int256 accruedFunding;
-        uint256 pnlUint;
-        uint256 amountToDeduct;
```

Comment:
@leomassazza just to verify here, these are not used anymore, should be safe to remove right since this struct is only instantiated during runtime?

Reply:
no problem here, the issue is for storage structs, not runtime.